moped-database/metadata/tables.yaml

```diff
@@ -1,16 +1,16 @@
 - table:
-    schema: public
     name: moped_activity_log
+    schema: public
```

Comment:
it bums me out when it changes the order thus making a lot of nonchanges

Reply:
ugh, yes. i thought we were in a better place now. should i make a clean diff for y'all?